src/H5FDprivate.h

```diff
@@ -39,6 +39,220 @@
 /* Length of filename buffer */
 #define H5FD_MAX_FILENAME_LEN 1024
 
+/*
```

Comment:
All of this should be moved to the H5FDvfd_swmr.h source file (or header, if the tests need any of it).